TaskManagement/src/TaskManagment/DB methods/Connections.java

```diff
@@ -0,0 +1,17 @@
+package TaskManagerProject;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Connections {
+    public static java.sql.Connection getConnection() {
+        try {
+            java.sql.Connection con = DriverManager
+                    .getConnection("jdbc:mysql://localhost:3306/task_management_system?serverTimezone=UTC", "root", "password");
```

Comment:
The database URL, username, and password are hard-coded within the getConnection() method. This can be problematic because it makes it difficult to change the connection details without modifying the source code. Consider using external configuration files, environment variables, or dependency injection to provide the connection details externally.

Reply:
@HadeelSawalha Let's store the the db name, db port, username, password, etc. as environment variables. And if you plan to deploy the database then it's better to make a separate env file for each environment: DEVELOPMENT & PRODUCTION